errors/IncorrectAnonymousArgument.md

```diff
@@ -5,31 +5,54 @@
 ```purescript
 module Example where
 
-add = (_ + _)
+sub3 = _ - 3
 
-mapArray = map _ [1, 2, 3]
```

Comment:
I think this second-order example adds another layer of potential confusion for beginners. I changed all these `map` examples to use first-order arguments for `_`.